psi4/src/psi4/libmints/mintshelper.cc

```diff
@@ -894,34 +905,96 @@ SharedMatrix MintsHelper::ao_eri_shell(int M, int N, int P, int Q) {
 }
 
 SharedMatrix MintsHelper::ao_erfc_eri(double omega) {
-    std::shared_ptr<TwoBodyAOInt> ints(integral_->erf_complement_eri(omega));
-    return ao_helper("AO ERFC ERI Tensor", ints);
+    std::vector<std::shared_ptr<TwoBodyAOInt>> ints_vec(nthread_);
+    ints_vec[0] = std::shared_ptr<TwoBodyAOInt>(integral_->erf_complement_eri(omega));
+    for (size_t i = 1; i < nthread_; i++) {
+        ints_vec[i] = std::shared_ptr<TwoBodyAOInt>(ints_vec.front()->clone());
+    }
+    return ao_helper("AO ERFC ERI Tensor", ints_vec);
 }
 
 SharedMatrix MintsHelper::ao_f12(std::vector<std::pair<double, double>> exp_coeff) {
-    std::shared_ptr<TwoBodyAOInt> ints(integral_->f12(exp_coeff));
-    return ao_helper("AO F12 Tensor", ints);
+    std::vector<std::shared_ptr<TwoBodyAOInt>> ints_vec(nthread_);
+    ints_vec[0] = std::shared_ptr<TwoBodyAOInt>(integral_->f12(exp_coeff));
+    for (size_t i = 1; i < nthread_; i++) {
+        ints_vec[i] = std::shared_ptr<TwoBodyAOInt>(ints_vec.front()->clone());
+    }
```

Comment:
This might actually be better to run in parallel. That way the memory can get allocated to the processor's local memory.